common/app/conf/GoogleAuth.scala

```diff
@@ -0,0 +1,52 @@
+package conf
+
+import conf.Configuration.OAuthCredentials
+import com.amazonaws.auth.AWSCredentialsProviderChain
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagementClientBuilder
+import com.gu.googleauth.{AntiForgeryChecker, GoogleAuthConfig}
+import com.gu.play.secretrotation.{SnapshotProvider, TransitionTiming}
+import play.api.http.HttpConfiguration
+
+import java.time.Duration.{ofHours, ofMinutes}
+
+case class GoogleAuth(
+    currentHost: Option[String],
+    httpConfiguration: HttpConfiguration,
+    oauthCredentials: Option[OAuthCredentials],
+) {
+  private val securityCredentialsProvider = new AWSCredentialsProviderChain(
```

Comment:
I think the naming of this `val` might be a carry over from security HQ? We called them security credentials because they are credentials to the security AWS account - you might want to rename.

Reply:
Indeed! Renamed to `frontendCredentialsProvider`.